File: Project-Euler/Solutions-Simon/euler7/euler7.py
def isPrime(x):
	divisor = 2
	prime = True
	while divisor < x:
		if x % divisor == 0:
			prime = False
			break
		divisor += 1
	return prime
	
def findPrime():
	arrayPrimes = []
	counter = 0
	number = 2
	primeNumber = 10001
	while counter < primeNumber:
		if isPrime(number):
			counter += 1
			if counter == primeNumber:
				break
    		number += 1
	arrayPrimes.append(number)
	print arrayPrimes[-1]

findPrime()

File: Project-Euler/Solutions-Simon/euler7/euler7.cpp
//
//  main.cpp
//  Euler7
//
//  Created by Sonni on 19/05/16.
//  Copyright © 2016 Sonni Jensen. All rights reserved.
//

#include <iostream>
#include <vector>

bool isPrime(unsigned int x)
{
    unsigned int divisor = 2;
    bool prime = true;
    while (divisor < x)
    {
        if (x % divisor == 0)
        {
            prime = false;
            break;
        }
        divisor++;
    }
    return prime;
}

void findPrime(unsigned int primeNumber)
{
    std::vector<unsigned int> arrayPrimes;
    unsigned int counter = 0;
    unsigned int number = 2;
    
    while (counter < primeNumber)
    {
        if (isPrime(number))
        {
            counter++;
            if (counter == primeNumber)
                break;
        }
        number++;
    }
    arrayPrimes.push_back(number);
    std::cout << arrayPrimes[arrayPrimes.size()-1] << std::endl;
}

int main(int argc, const char * argv[]) {
    findPrime(atoi(argv[1]));
    return 0;
}
